common/commands/src/validator/cosmwasm/execute_contract.rs

```diff
@@ -0,0 +1,41 @@
+// Copyright 2021 - Nym Technologies SA <contact@nymtech.net>
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::context::SigningClient;
+use clap::Parser;
+use cosmrs::AccountId;
+use log::{error, info};
+use serde_json::{json, Value};
+
+#[derive(Debug, Parser)]
+pub struct Args {
+    #[clap(value_parser)]
+    #[clap(help = "The address of contract to execute")]
+    pub contract_address: AccountId,
+
+    #[clap(value_parser)]
+    #[clap(help = "JSON encoded method arguments")]
+    pub json_args: String,
+
+    #[clap(long)]
+    pub memo: Option<String>,
```

Comment:
We also need an extra argument here for `funds`. For example you wouldn't be able to bond a mixnode with `nym-cli` in its current state.

Reply:
Added that as `funds` and `funds_denom`